extensions/tree-inspector/tree-inspector.js

```diff
@@ -399,6 +413,23 @@ var TreeInspector = (function() {
         updateCheckboxControl.appendChild(createElement("label", "Update When Debugger Breaks", {
             "for": id("RefreshOnBreak")
         }));
+        this.refreshOnBreakAndContinueGroupElement = createElement("span", null, { 
+            class: (this.refreshOnBreak ? "show":"hide"), 
+            id: id("RefreshOnBreakAndContinueGroup") 
+        });
+        this.refreshOnBreakAndContinueGroupElement.appendChild(createElement("input", null, {
+            name: "refreshOnBreakAndContinue",
+            id: id("RefreshOnBreakAndContinue"),
+            type: "checkbox",
+        }, {
+            "change": function (e) {
+                that.setRefreshOnBreakAndContinue(e.target.checked);
+            }
+        }));
+        this.refreshOnBreakAndContinueGroupElement.appendChild(createElement("label", "Continue execution", {

```

Comment:
Instead of "Continue execution". can we say "Continue after next break"? Because the debugger doesn't automatically continue after checking that box.

Reply:
"Continue after next break " doesnt explain behavior at each break. Continue execution is what visual studio calls it and the check box doesnt continue execution when you are currently broken in over there as well.

Reply:
@sushraja-msft I agree that "Continue execution" is what the VS debugger calls it, but as someone who doesn't use that debugger by default, it wasn't intuitive. I expected "Continue execution" do actually perform a continue from my current bp. Not going to block on this, but it'd be great if we can come up with a better name for this concept.